tests/unit/payments.js

```diff
@@ -165,6 +155,24 @@ define('tests/unit/payments',
             });
         });
 
+        it('defines a configurable adapter', function(done) {
+            var adapterConfigured = false;
+
+            setupMocks().mock('fxpay', {
+                init: function(opt) {
+                    // This simulates how fxpay will configure
+                    // the Marketplace adapter when fxpay.init()
+                    // is called.
+                    opt.adapter.configure({});
```

Comment:
Do you need to stub `fxpay` if you put a spy on `opt.adapter.configure` and check if it was called? If it's possible I'd suggest a spy regardless.